conf/cmi/webform.webform.nuorisotoiminta_projektiavustush.yml

```diff
@@ -226,8 +226,8 @@ elements: |-
         '#title': 'Valitse toiminnasta vastaavat henkilöt'
         '#multiple': true
         '#multiple__item_label': henkilö
-        '#multiple__min_items': 1
-        '#multiple__empty_items': 0
+        '#multiple__min_items': 0
+        '#multiple__empty_items': 1
```

Comment:
Nää varmaan vahingossa muuttunu, kun Petri tossa aikasemmin muutti ne min: 1?

https://github.com/City-of-Helsinki/hel-fi-drupal-grants/commit/c61b14ed4e5c5f3e100e00c8d97fa98d40fded5c#diff-8898281812c8e3cd176545a1572e1d810658716e092a735bde4e23594140cc05L229